backend/server.py

```
@@ -5628,6 +5648,30 @@ async def health():
+    fired = []
+    try:
+        jobs = scheduler.list()
+        import datetime
+        now = datetime.datetime.utcnow()
+        for job in jobs:
+            next_run = job.next_run_time if hasattr(job, "next_run_time") else None
+            if next_run and next_run <= now:
+                try:
+                    scheduler.trigger(job.job_id)
+                    fired.append(job.job_id)
+                except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)